### PR TITLE
[terra-core] Fix broken links in docs

### PR DESCRIPTION
**Pull request by saket2403**
### Summary
fixed broken links in docs.

### Deployment Link

https://terra-core-deployed-pr-#.herokuapp.com/

### Additional Details

Thank you for contributing to Terra.
@cerner/terra

### CODE REVIEW

**Review by emilyrohrbough — APPROVED** [review]

**Review by supreethmr — APPROVED** [review]

**Review by StephenEsser — APPROVED** [review]

**Comment by mjhenkes** [comment]
@saket2403 this one is ready to merge after you fix the merge conflicts.

**Pull request merged**
